### DISCUSSION TITLE
Documentation typo in Youtube extension

### DISCUSSION THREAD

**Discussion by GiovanniSalmeri** [datenstrom/community#860] [Report a bug] [answered]
In https://github.com/annaesvensson/yellow-youtube there is the example:

```
[youtube fhs55HEl-Gc right 50%]
```

If I am not wrong, that `50%` alone has no effect (`if ($width && $height) ...`). Or perhaps is it the code that should be modified? Thank you very much!

**Comment by markseuffert** [comment, marked as answer]
Thank you for the bug report. After checking the file history I found out what happened. The documentation was updated, but the code was not. I modified the source code to work with the given examples. Should be fixed now, please try it out.

**Reply by GiovanniSalmeri** [reply]
All works perfectly, thank you very much!

Just out of curiosity: is there a specific reason for considering 640x360 the 100% dimension (in lines 23-24)? The default embed dimension for YouTube is currently slightly smaller, i.e. 560x315. (As a sidenote, if a video is embedded with an iframe without dimensions, it defaults to 300x150, but this is the default dimension of an `iframe`, not of a YouTube video.)

**Reply by markseuffert** [reply]
It's a guesstimate, since we don't know the exact dimensions of the original video. YouTube thumbnails have an image resolution of 1280x720 pixels, sometimes known as HDTV resolution. Should we take this instead?

**Reply by GiovanniSalmeri** [reply]
I think it is perfect as it is now, 640x360 is a very sensible size. Thank you again!
